doc-content/enterprise-only/release-notes/rn-730-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,149 @@
+//[https://issues.jboss.org/browse/RHDM-827[RHDM-827]]
+
+
+== Decision engine
+
+* Unexpected node sharing by function with equals [https://issues.jboss.org/browse/RHDM-850[RHDM-850]]
+* From with modify fires unexpected rule [https://issues.jboss.org/browse/RHDM-841[RHDM-841]]
+* Misleading error messagaes generated by maven plugin when creating an executable model kjar [https://issues.jboss.org/browse/RHDM-843[RHDM-843]]
+* Compiler error in executable model when there is a clash between a field and a class name [https://issues.jboss.org/browse/RHDM-834[RHDM-834]]
+* "NoSuchElementException: No value present" with executable-model, Map and binding [https://issues.jboss.org/browse/RHDM-832[RHDM-832]]
+* The behavior of `update` is different from the behavior of `modify` when the the `mvel` dialect is used with property reactive type [https://issues.jboss.org/browse/RHDM-830[RHDM-830]]
+* The executable model fails to compare BigDecimal [https://issues.jboss.org/browse/RHDM-824[RHDM-824]]
+* kbase packages in the `kmodule.xml` file work differently with the executable model [https://issues.jboss.org/browse/RHDM-823[RHDM-823]]
+* In the executable model, a build error occurs when multiple DRLs of the same package are used [https://issues.jboss.org/browse/RHDM-819[RHDM-819]]
+* In the executable model, a parse error occurs is you use the Java dialect and connect expressions with a inside a modify block [https://issues.jboss.org/browse/RHDM-818[RHDM-818]]
+* The MVEL expression `(1 + 2 * 3 + 4 * $v )` causes a `no such method or function` error for bind variables [https://issues.jboss.org/browse/RHDM-815[RHDM-815]]
+* A rule with many accumulates is corrupted by `DrlParser` and `DrlDumper` [https://issues.jboss.org/browse/RHDM-811[RHDM-811]]
+Compiler error in executable model when there is a clash between a field and a class name
+* In Windows, if you run `CRLF` in a `.xlsx` spreadsheet ACTION cell, the value is not treated as an absolute value [https://issues.jboss.org/browse/RHDM-807[RHDM-807]]
+
+==  Data modeler
+
+* When creating a data object class that implements the Comparable interface, a `ClassNotFoundException` is thrown when saving the asset [https://issues.jboss.org/browse/RHDM-734[RHDM-734]]
+
+== {KIE_SERVER}
+
+* The thread used by `LoadBalancer` remains when the connection to {KIE_SERVER} fails during the instantiation of `KieServicesClient` [https://issues.jboss.org/browse/RHDM-829[RHDM-829]]
+* Beans annotated with `@propertyChangeSupport` do not account for the property reactvity when propagating a modification  [https://issues.jboss.org/browse/RHDM-846[RHDM-846]]
+
+== OpenShift
+
+* {CENTRAL} liveness and readiness probes point to the wrong URL [https://issues.jboss.org/browse/RHDM-847[RHDM-847]]
+* The `rhpam70-authoring-ha` template does not configure AMQ to be high availability [https://issues.jboss.org/browse/RHDM-861[RHDM-861]]
```

Comment:
Update to "The `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml` template ..." so it renders correctly in both products.